app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -122,6 +124,34 @@ const MeetingEditor = () => {
       : undefined,
   );
 
+  //For some reason calling the debounce function directly was wack.
+  //Also, we want to check if the new length is greater than the old one or if it doesn't exist already.
+  const updateReportValue = useCallback(
+    (newReportValue) => {
+      const debounceFunc = debounce(() => {
+        const storedReport = sessionStorage.getItem(
+          `meeting-${meetingId}-report`,
+        );
+        if (!storedReport || storedReport.length < newReportValue.length) {
+          setReportValue(newReportValue);
+        }
+      }, 4500);
+      debounceFunc();
+    },
+    [meetingId],
+  );
+
+  useEffect(() => {
+    if (meeting && meetingId) {
+      const storedReport = sessionStorage.getItem(
+        `meeting-${meeting?.id}-report`,
+      );
+      if (!storedReport || storedReport.length < reportValue.length) {
+        sessionStorage.setItem(`meeting-${meeting?.id}-report`, reportValue);
```

Comment:
Minor/unlikely security concern. A user who has signed out after editing a report will still have this data in their session storage. A quick fix here is clearing it whenever a user signs out, i.e. add a `sessionStorage.clear()` [here](https://github.com/webkom/lego-webapp/blob/master/app/actions/UserActions.ts#L88)

Reply:
You can just clear it when this component unmounts

Reply:
Why add it to an action that has nothing to do with this?